src/autopas/containers/verletClusterLists/VerletClusterCells.h

```diff
@@ -433,19 +440,19 @@ class VerletClusterCells : public ParticleContainer<FullParticleCell<Particle>>
       const auto numParticles = this->_cells[i].numParticles();
 
       _dummyStarts[i] = numParticles;
-      unsigned int numDummys = _clusterSize;
-      if (numParticles > 0) {
-        numDummys -= (numParticles % (size_t)_clusterSize);
-      }
-
-      Particle dummyParticle = Particle();
-      for (unsigned int j = 0; j < numDummys; ++j) {
-        dummyParticle.setR({_boxMaxWithHalo[0] + 8 * this->getInteractionLength() + static_cast<double>(i),
-                            _boxMaxWithHalo[1] + 8 * this->getInteractionLength() + static_cast<double>(j),
-                            _boxMaxWithHalo[2] + 8 * this->getInteractionLength()});
-        dummyParticle.setID(std::numeric_limits<size_t>::max());
-        dummyParticle.setOwned(false);
-        this->_cells[i].addParticle(dummyParticle);
+      auto sizeLastCluster = (numParticles % _clusterSize);
+      unsigned int numDummys = sizeLastCluster != 0 ? _clusterSize - sizeLastCluster : 0;
```

Comment:
this changes the behavior a bit:
now no dummy particles are generated if a cell contains no particles.